elements/rh-skip-link/rh-skip-link.css

```diff
@@ -0,0 +1,62 @@
+#rhds-skip-link-container {
+  display: block !important;
+  position: relative !important;
+}
+
+::slotted(a) {
+  background-color: var(--rh-color-surface-lightest, #ffffff) !important;
+  border-block-start-width: 0 !important;
+  border-block-end:
+    var(--rh-border-width-sm, 1px)
+    solid
+    var(--rh-color-border-interactive-on-light, #0066cc) !important;
+  border-inline-start:
+    var(--rh-border-width-sm, 1px)
+    solid
+    var(--rh-color-border-interactive-on-light, #0066cc) !important;
+  border-inline-end:
+    var(--rh-border-width-sm, 1px)
+    solid
+    var(--rh-color-border-interactive-on-light, #0066cc) !important;
+  border-end-start-radius: var(--rh-border-radius-default, 3px) !important;
+  border-end-end-radius: var(--rh-border-radius-default, 3px) !important;
+  clip: rect(0, 0, 0, 0) !important;
+  color: var(--rh-color-interactive-blue-darker, #0066cc) !important;
+  font-family: var(--rh-font-family-heading, RedHatDisplay, "Red Hat Display", "Noto Sans Arabic", "Noto Sans Hebrew", "Noto Sans JP", "Noto Sans KR", "Noto Sans Malayalam", "Noto Sans SC", "Noto Sans TC", "Noto Sans Thai", Helvetica, Arial, sans-serif) !important;
+  font-weight: var(--rh-font-weight-heading-bold, 700) !important;
```

Comment:
Can `font-size: var(--rh-font-size-body-text-md, 1rem);` be added to make sure page styles don't change that too?

Reply:
Fixed in https://github.com/RedHat-UX/red-hat-design-system/pull/1515/commits/26ce8e3c5a16da825537fad19292c0e0f21346ed. 

Not sure why GitHub still marks this as "1 change requested"? 

@marionnegp perhaps you can mark this as resolved or hit "Dismiss Review". TIA.